spynnaker_integration_tests/test_various/test_models.py

```diff
@@ -0,0 +1,37 @@
+
+# Copyright (c) 2017-2019 The University of Manchester
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import pyNN.spiNNaker as sim
+from spinnaker_testbase import BaseTestCase
+
+
+class TestPopulation(BaseTestCase):
+
+    # NO unittest_setup() as sim.setup is called
```

Comment:
Probably no need to have this comment here as this is in the integration tests, not the unit tests.

Reply:
The integration tests use this also, but I can see it is confusing as a comment!